### PR TITLE
train_paraphraser_model is implemented in wscript

### PR DESCRIPTION
**Pull request by nikolay-bushkov**

### CODE REVIEW

File: utils/model_tests.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import unittest
+import train_model as model
+
+class KPIException(Exception):
```
Comment (bavadim) [inline utils/model_tests.py]: Этот класс к тестам отношения не имеет и не должен тут лежать

File: utils/model_tests.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    """Class for exceptions raised if some KPI is not satisfied"""
+    pass
+
+
+class KPITests(unittest.TestCase):
+    """Class for tests of different KPIs"""
+
+    def test_paraphraser(self):
+        metric = model.main(['-t', 'deeppavlov.tasks.paraphrases.agents',
```
Comment (bavadim) [inline utils/model_tests.py]: это не metric, а fscore

File: utils/model_tests.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                         '--learning_rate', '0.0001',
+                         '--hidden_dim', '200',
+                         '--validation-every-n-epochs', '5',
+                         '--fasttext_embeddings_dict', '"./build/paraphraser.emb"',
+                         '--fasttext_model', '"./build/ft_0.8.3_nltk_yalen_sg_300.bin"',
+                         '--cross-validation-seed', '50',
+                         '--cross-validation-splits-count', '5',
+                         '--validation-patience', '3',
+                         '--chosen-metric', 'f1'
+#,                         '--pretrained_model', './build/paraphraser'
```
Comment (bavadim) [inline utils/model_tests.py]: это что за параметр? Если не нужен - удали

File: utils/model_tests.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+	])
+    
+        self.assertTrue(metric['f1'] > 0.8)
```
Comment (bavadim) [inline utils/model_tests.py]: msg укажи

File: utils/train_model.py [hunk 5]
```diff
@@ -203,24 +203,26 @@ def train_model(opt):
...
 
 
 def train_cross_valid(opt):
     if opt.get('model_files'):
         opt['model_files'] = [fname+'_'+str(i) for fname in opt['model_files']
                               for i in range(opt['cross_validation_splits_count'])]
-        train_model(opt)
-        return
+        return train_model(opt)
+    metric = {'f1': 0, 'accuracy': 0}
```
Comment (bavadim) [inline utils/train_model.py]: пусть это будет None

File: wscript [hunk 6]
```diff
@@ -1,10 +1,49 @@
 import os
+import utils.train_model as model
+from utils.model_tests import KPIException
```
Comment (bavadim) [inline wscript]: класс эксепшна пусть будет в этом файле

File: wscript [hunk 7]
```diff
@@ -1,10 +1,49 @@
...
 
 def train_paraphraser_model(ctx):
-  pass
+
+    metric = model.main(['-t', 'deeppavlov.tasks.paraphrases.agents',
```
Comment (bavadim) [inline wscript]: нормальное название переменной дай, типа fscore

File: wscript [hunk 8]
```diff
@@ -1,10 +1,49 @@
...
+                         '--fasttext_embeddings_dict', '"./build/paraphraser.emb"',
+                         '--fasttext_model', '"./build/ft_0.8.3_nltk_yalen_sg_300.bin"',
+                         '--cross-validation-seed', '50',
+                         '--cross-validation-splits-count', '5',
+                         '--validation-patience', '3',
+                         '--chosen-metric', 'f1'
+#,                         '--pretrained_model', './build/paraphraser'
+	])
+
+    return
```
Comment (bavadim) [inline wscript]: это не нужно

File: wscript [hunk 9]
```diff
@@ -1,10 +1,49 @@
...
 
 def test_paraphraser_model(ctx):
```
Comment (bavadim) [inline wscript]: тут давай вызывать unittest. Погугли как вызвать unittest из waf
